feat: Пеплоходцы и риги

definitelynotspaghetti opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->
Пеплоходцы больше не могут носить РИГи, костюмы ЕВА. При попытке поднять риг он улетит в случайном направлении на 2-5 тайлов.
## Ссылка на предложение/Причина создания ПР
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->
Авоут пройден давно.
https://discord.com/channels/617003227182792704/755125334097133628/1043845128802021447
## Демонстрация изменений
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

definitelynotspaghetti committed:
ash walkers

definitelynotspaghetti committed:
bleh

definitelynotspaghetti commented:
![bleh](https://user-images.githubusercontent.com/48985748/211212811-538d7e62-ed9f-4c92-bddb-806fa72d3721.jpg)

Jerrody commented:
Можно я не буду это комментировать?

Honker9999999 commented:
Я, офк, не разраб и не контрибубор, но по моему отлетание на 2-4 тайла излишне и по предложке не прошло.

"Запретить кодом/механом возможность Пеплоходцам надевать на себя **любые** типы ригов(хардсьютов) и костюмов ВКД(В простонародье именуемые Евой) по аналогии с запретом, присутствующим на КА/ЕКА. 
Запрет заключается в невозможности Пеплоходцу нацепить на себя риг/костюм ВКД. "

Honker9999999 reviewed (changes requested):
Добавь проверку на стан/паралич/отсутствие рук у эша. На данный момент `/obj/item/clothing/suit/equipped` проигрывается в том числе если одежду надел кто-то иной, из-за чего выходит что эш каким-то образом откинет скаф даже если он в стане/имеет отрезанные руки или вообще умер.
Кроме того, ты забыл` ..()`, из-за чего родительский код equipped не проигрывается, что потенциально сломает прятание хвостов при одевании скафандров вообще у всех.

definitelynotspaghetti committed:
finish

equipped() странно работает, поэтому заменмл прок на mob_can_equip(). Ну и убрал отбрасывание

Honker9999999 commented:
Чзх тут только что произошла

definitelynotspaghetti commented:
ничего
